db/migrate/20180322085635_create_reviews.rb

```diff
@@ -0,0 +1,14 @@
+class CreateReviews < ActiveRecord::Migration[5.1]
+  def change
+    create_table :reviews do |t|
+      t.string :reviewer_name
+      t.integer :rating
+      t.text :comment
+      t.integer :restaurant_id, null: false
```

Comment:
You can also write this as:
```ruby
t.references :restaurant, null: false # [, index: true, foreign_key: true]
```

https://apidock.com/rails/ActiveRecord/ConnectionAdapters/Table/references

Reply:
:+1: thanks